contracts/src/integrations/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -35,10 +36,17 @@ contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgrade
 
     /// ADMIN ///
 
-    function initialize(address _initialOwner, address certVerifier) external initializer {
+    function initialize(address _initialOwner, uint32[] memory initRBNs, address[] memory initCertVerifiers)
```

Comment:
I think `initRBNs` should technically be `initABNs`, since that's the terminology used when tracking the verifier addresses in this contract

Reply:
https://github.com/Layr-Labs/eigenda/pull/1787/commits/f264b3f04cc48d9fba5bc86d37bdc7d4bd253178